libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectDistance.kt

```diff
@@ -0,0 +1,52 @@
+package com.mapbox.navigation.base.trip.model.roadobject
+
+/**
+ * RoadObjectDistance
```

Comment:
Could you elaborate here what the purpose of this class?

Reply:
done